example.env

```diff
@@ -0,0 +1,40 @@
+# Example of what a .env file could look like
```

Comment:
Example of env variables that can be set when running manually, with some explanations